subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/parser/ModuleMetadataParser.java

```diff
@@ -54,7 +54,7 @@
 import static org.apache.commons.lang.StringUtils.capitalize;
 
 public class ModuleMetadataParser {
-    public static final String FORMAT_VERSION = "0.4";
+    public static final String FORMAT_VERSION = "1.0";
```

Comment:
I think we should also change the parser so that it can deal with future changes that we make. For example, it might accept any 1.x version and ignore things it does not understand in the metadata.

We could also perhaps think about a way to retire 1.0 in the future. For example, we could have the parser fail on any version that isn't 1.x and just start generating 2.0 metadata when we want to make a breaking change. Or perhaps we just aim to never break 1.0.

Reply:
I have made the parser lenient. If an error occurs, and that we didn't recognize the version, it will indicate it. Otherwise regular errors are thrown.